### PR TITLE
Additional resource to Frontend Project

### PR DESCRIPTION
**Pull request by muhammadiqbalid83**

### CODE REVIEW

**Review by shivaabhishek07 — COMMENTED** [review]
What additional resources you have added?

**Comment by muhammadiqbalid83** [comment]
this : https://devchallenges.io/

**Comment by shivaabhishek07** [comment]
> this : https://devchallenges.io/

Ok!

File: Frontend-Development/README.md [hunk 1]
```diff
@@ -32,11 +42,14 @@
 - [HTML Tutorial, making a Website - FreeCodeCamp(Youtube)](https://www.youtube.com/watch?v=PlxWf493en4)
 
 ### Frontend Projects
+
 - [frontendmentor.io](https://www.frontendmentor.io/)
 - [frontendpractice.com](https://www.frontendpractice.com/)
 
 ## Learning CSS
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:50]: remove whitespace ONLY for line 50

File: Frontend-Development/README.md [hunk 2]
```diff
@@ -45,11 +58,14 @@
 - [Introduction to CSS - Web Dev Simplified](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH9D0J0Mtp6lIiD_8046k3si)
 
 ### Frontend Projects
+
 - [frontendmentor.io](https://www.frontendmentor.io/)
 - [frontendpractice.com](https://www.frontendpractice.com/)
 
 ## Learning Javascript
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:66]: Remove whitespace ONLY for like 66

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Please resolve the lint error

**Comment by siddhant-khisty** [comment]
Closing due to inactivity

**Pull request closed without merging**